plugins/plugin-python/che-plugin-python-lang-server/pom.xml

```diff
@@ -64,5 +60,13 @@
             <groupId>org.eclipse.che.plugin</groupId>
             <artifactId>che-plugin-python-lang-shared</artifactId>
         </dependency>
+		<dependency>
```

Comment:
I know, but doing the above DOES NOT CHANGE ANYTHING except screw up some formatting.This pom already seems sorted.